src/main/kotlin/com/terraformation/backend/accelerator/db/CohortStore.kt

```diff
@@ -118,6 +116,31 @@ class CohortStore(
         throw CohortNotFoundException(cohortId)
       }
 
+      if (existing.modules != updated.modules) {
+        val toRemove =
+            existing.modules.map { it.moduleId }.toSet() -
+                updated.modules.map { it.moduleId }.toSet()
+
+        with(COHORT_MODULES) {
+          toRemove.forEach { dslContext.deleteFrom(this).where(MODULE_ID.eq(it)).execute() }
+
+          updated.modules.forEach {
+            dslContext
+                .insertInto(this)
+                .set(COHORT_ID, cohortId)
+                .set(MODULE_ID, it.moduleId)
+                .set(TITLE, it.title)
+                .set(START_DATE, it.startDate)
+                .set(END_DATE, it.endDate)
+                .onDuplicateKeyUpdate()
+                .set(TITLE, it.title)
+                .set(START_DATE, it.startDate)
+                .set(END_DATE, it.endDate)
+                .execute()
+          }
+        }
+      }
```

Comment:
This feels like it belongs in another PR -- it has nothing to do with the GET endpoint, right?